src/cmd/builtin/version.c

```diff
@@ -26,9 +26,29 @@
 
 #include "builtin.h"
 
+
+static void print_broker_version (optparse_t *p)
+{
+    const char *uri = getenv ("FLUX_URI");
+    const char *version;
+
+    if (!uri)
+        return;
+    flux_t *h = builtin_get_flux_handle (p);
+    if (!h)
+        log_err_exit ("flux_open");
+    if (!(version = flux_attr_get (h, "version", NULL)))
```

Comment:
Rather than exiting with error on an invalid `FLUX_URI`, would it be better to either
 - silently ignore error from `flux_open` and act as if `FLUX_URI` wasn't set
 - or report a more user-friendly error `Failed to connect to broker at FLUX_URI=%s`
?

Reply:
I think at minimum a user friendly error should be reported since other things aren't going to do well if you have FLUX_URI set to a dead instance or whatever.

Reply:
Thanks, `flux start -s` will still work, but going forward that will not be a very common case, so I agree with your assesment...